auction-server/src/opportunity/service/verification.rs

```diff
@@ -279,3 +285,16 @@ impl Service<ChainTypeEvm> {
         }
     }
 }
+
+impl Verification<ChainTypeSvm> for Service<ChainTypeSvm> {
+    async fn verify_opportunity(
+        &self,
+        input: VerifyOpportunityInput<entities::OpportunityCreateSvm>,
+    ) -> Result<entities::OpportunityVerificationResult, RestError> {
+        self.get_config(&input.opportunity.core_fields.chain_id)?;
+
+        // To make sure it'll be expired after a minute
+        // TODO - change this to a more realistic value
+        Ok(entities::OpportunityVerificationResult::UnableToSpoof)
```

Comment:
maybe `UnableToSpoof` should have a u256 param representing number of seconds this opp can be present? Can set then set this to different values, don't need to check staleness agaisnt `MAX_STALE_OPPORTUNITY_MICROS`

Reply:
I'm not sure if expiring a opportunity is the verification responsibility or not :-?